rcl/test/rcl/test_graph.cpp

```diff
@@ -274,7 +274,7 @@ TEST_F(
   rcl_allocator_t zero_allocator = static_cast<rcl_allocator_t>(
     rcutils_get_zero_initialized_allocator());
   rcl_node_t zero_node = rcl_get_zero_initialized_node();
-  const char * unknown_node_name = "/test_rcl_get_publisher_names_and_types_by_node";
+  const char * unknown_node_name = "test_rcl_get_publisher_names_and_types_by_node";
```

Comment:
Is this change necessary? I believe the test is for an *unknown* node, not an invalid node name.
Same for similar changes to below.

Reply:
Node names can't start with `/`, namespaces can.
If I left the `/` at the start, I will get an invalid name error, and that's not the idea.

Reply:
Ok, got it.